src/routes/api/api-router.ts

```diff
@@ -138,7 +139,8 @@ ApiRouter.use("/cryptor", async (_req: Request, resp: Response) => {
 		resp.status(500).send("fail");
 	}
 });
+ApiRouter.use("/migration", CryptorMigrationRouter);
 
 ApiRouter.use("/jira", ApiJiraRouter);
+//TODO: Following `use(/:installationId)` seems block any more routers to be used bellow, need fix
```

Comment:
errant comment?

Reply:
...yes, that's normal.  that's how routing works

Reply:
Okay if you believe it is normal to prevent more routes underneath. We can always come back to this if need to.
For now I removed the comments per your suggestions. Resolve this convo now.